components/function-controller/cmd/webhook/main.go

```diff
@@ -138,3 +141,25 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func configureLogger(ctx context.Context, cfgPath string) (*logging.Registry, error) {
```

Comment:
This function is doing two things:
-  reading config
- configure logging with this config

If we split it into two functions we can move the configuration of logger to `internal/logging` to remove duplicate.